docs/content/guides/major-pg-upgrade.md

```diff
@@ -0,0 +1,242 @@
+        resources:
+          requests:
+            storage: 1Gi
+```
+
+You will need to set it to something similar to 
+
+```
+spec:
+  instances:
+    - name: instancea
+      replicas: 1
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteMany"
+        resources:
+          requests:
+            storage: 1Gi
+```
+
+then apply the changes with
+
+```
+kubectl apply -k examples/postgrescluster
+```
+
+and wait for them to complete before continuing on to the next steps.
+
+<!-- TODO(tjmoore4): This step should not be required after follow on work to run pgBackRest stanza 
+upgrade and backup automatically during upgrade process. This will allow the replicas to be automatically
+scaled down before the upgrade and back up once the post-upgrade steps are completed. -->
+
+### Step 3: Configure the PostgresCluster
+
+At this point, your running cluster is ready for the major upgrade. As shown below, you will need to 
+configure the `spec.upgrade` section to enable the major upgrade. This section will also define the current 
+(i.e. pre-upgrade) Postgres version of the cluster and define the `crunchy-upgrade` image, assuming the
+relevant environment variable, `RELATED_IMAGE_PGUPGRADE`, is not being utilized. 
+
+At the same time, you will need to set the `spec.postgresVersion` and `spec.image` values to reflect the
+version the PostgresCluster will be after the upgrade is completed. Below, you'll see an example configuration
+for a {{< param fromPostgresVersion >}} to {{< param postgresVersion >}} upgrade:
+
+```
+spec:
+  upgrade:
+    enabled: true
+    fromPostgresVersion: {{< param fromPostgresVersion >}}
+    image: {{< param imageCrunchyPGUpgrade >}}
+  image: {{< param imageCrunchyPostgres >}}
+  postgresVersion: {{< param postgresVersion >}}
+```
+
+Please note, the `spec.upgrade` section can be set in advance of running the upgrade, just be sure 
+to set `enabled` to `false`. 
+
+{{% notice warning %}}
+Setting and applying the `postgresVersion` or `image` values before configuring and enabling the
+`spec.upgrade` section will result in an error due to the PostgreSQL version mismatch!
+{{% /notice %}}
+
+### Step 4: Run the PostgreSQL Upgrade
+
+Once everything is configured as described above, the upgrade can be started by running
+
+```
+kubectl apply -k examples/postgrescluster
+```
+
+This will then terminate the database instance and start the pg_upgrade Job. This Job will perform the
+necessary steps to upgrade the PostgresCluster to the desired version. Once this Job completes, the
+primary instance will then be started and should return to the previous running state.
+
+### Step 5: Remove the Upgrade Configuration
+
+Once the upgrade Job has completed and the primary instance is running, the upgrade section configured above, i.e.
+
+```
+spec:
+  upgrade:
+    enabled: true
+    fromPostgresVersion: {{< param fromPostgresVersion >}}
+    image: {{< param imageCrunchyPGUpgrade >}}
+```
+
+should be removed.
```

Comment:
I wouldn't split the subject from the verb by the example yaml. (I'd put the example yaml after, or maybe leave it out entirely?)